src/client/app/components/about.ts

```diff
@@ -15,15 +15,15 @@ let { config } = PLAYGROUND;
                     <div class="profile__tertiary-text ms-font-m">User ID: ${storage.user}</div>
                     <div class="about__secondary-text ms-font-l">Version: {{config?.build?.version}}
                         <br/><span class="ms-font-m">(Deployed {{config?.build?.humanReadableTimestamp}})</span>
-                        <br/><span class="ms-font-m">{{config?.editorUrl}}</span>
                     </div>
-                    <div class="about__dropdown">
-                        <label class="ms-font-m about__dropdown-label">${Strings.aboutChangeEnvironment}</label>
-                        <select class="about__dropdown-select ms-font-m" [(ngModel)]="selectedConfig" (change)="changeConfig($event.target.value)">
+                    <pre class="about__tertiary-text ms-font-m">{{cache}}</pre>
```

Comment:
Screenshots have been updated in the pull request description. For easy reference, here's a picture of the about menu below:

![capture3](https://user-images.githubusercontent.com/14340560/27755797-f96e13b0-5da6-11e7-986f-97a483b82856.PNG)